### PR TITLE
Organize the presentations page, add content and update links

### PR DESCRIPTION
**Pull request by dorrab**
I have changed to division from recent/previous presentations to case studies/deep dives.
I think this is a better way to present the content.
I have added 2 recents talks by @mstoodle and @Leonardo2718
I have updated the broken links

### CODE REVIEW

File: resources/presentations.md [hunk 1]
```diff
@@ -23,22 +23,24 @@ lang: en
...
-- [*Building Your Own Runtime.*](https://ibm.box.com/s/7xdg25we2ezmdjjbqdys30d7dl1iyo49) Angela Lin, Robert Young, Craig Lehmann, and Xiaoli Liang, CASCON Workshop, November 2015.
-- [*What's in an Object? Java Garbage Collection for the Polyglot.*](http://www.slideshare.net/charliegracie1/javaone-whats-in-an-object) Charlie Gracie, Java One, October 2015.
-- [*Beyond the Coffee Cup: Leveraging Java Runtime Technologies for the Polyglot.*](http://www.slideshare.net/0xdaryl/javaone-2015-con7547-beyond-the-coffee-cup-leveraging-java-runtime-technologies-for-polyglot?related=1) Daryl Maier, Java One, October 2015.
-- [*A VM is a VM is a VM: The Secret Path to High Performance Multi-Language Runtimes.*](https://www.youtube.com/watch?v=kOnyJurioyw) Mark Stoodley, JVM Languages Summit, August 2015.
+- [*Testing Testarossa - The Eclipse OMR Compiler*](https://youtu.be/D0BCftV6DpE) Leonardo Banderali, August, 2017.
+- [*How does Eclipse OMR compare to LLVM*](http://www.ustream.tv/recorded/105013815) Mark Stoodley, June, 2017.
```
Comment (mstoodle) [inline resources/presentations.md]: Technically, the title was "Eclipse OMR: How does all this compare to LLVM?"

File: resources/presentations.md [hunk 2]
```diff
@@ -23,22 +23,24 @@ lang: en
...
+- [*Eclipse OMR Update Tech Talk*](http://www.ustream.tv/recorded/98846650) Mark Stoodley, developerWorks Open, January 2017.
```
Comment (mstoodle) [inline resources/presentations.md]: I know the talk didn't have a subtitle, but let's put one here to give people an idea what's in the presentation...Something like: "Eclipse OMR Update: compiler component, JitBuilder and projects that use them" ?

File: resources/presentations.md [hunk 3]
```diff
@@ -23,22 +23,24 @@ lang: en
...
+- [*Eclipse OMR Tech Talk*](https://developer.ibm.com/open/videos/eclipse-omr-tech-talk/) Mark Stoodley, developerWorks Open, July 2016.
```
Comment (mstoodle) [inline resources/presentations.md]: Do you have a link to the CUSEC presentation that Robert and Kim and John and Shelley went to? I can provide it if not; we should really add it in here too.

**Review by mstoodle — CHANGES_REQUESTED** [review]
Thanks for making this update, Dorra! Just a couple small nits to fix.

**Comment by dorrab** [comment]
Hello @mstoodle, I can't find the link. If you have it handy, please send it to me. Otherwise I can ask Shelley.

**Comment by mstoodle** [comment]
The presentation is here, I believe: https://github.com/youngar/Base9/blob/master/CUSEC%202017%20-%20Buildering%20a%20JIT.pdf

File: resources/presentations.md (hunk 3 above)
Comment (mstoodle) [reply resources/presentations.md]: Actually, I think it was Andrew <blush>, since Base9 is under his github id...

**Comment by dorrab** [comment]
Hello @mstoodle, addressed your comments above in the second commit. Please review. thank you

File: resources/presentations.md (hunk 3 above)
Comment (youngar) [reply resources/presentations.md]: Yeah, It was me and Kim who did the presentation

**Review by mstoodle — APPROVED** [review]

**Comment by mstoodle** [comment]
Thanks @dorrab!

**Pull request merged**